src/pmacfiltercontrol/pmacFilterControlWrapper.py

```diff
@@ -862,7 +862,9 @@ def _set_shutter_pos(self, val: float, shutter_state: str) -> None:
         if shutter_state == SHUTTER_CLOSED:
             self._configure_param({"shutter_closed_position": val})
 
-        current_shutter_state = "CLOSED" if self.shutter.get() == 0 else "OPEN"
```

Comment:
@GDYendell The reason I changed it to ints was because of this line. It would then pass a string into _set_shutter() which expects 0, otherwise the else is triggered. So pretty much it would always trigger the else clause as they were strings.

Reply:
I'm surprised mypy didn't notice the type issues

Reply:
I don't understand. You haven't changed the conditional, it is still checking `self.shutter.get() == 0`, you have only changed the assignment to be an int rather than a string, which just changes the value passed to `self._set_shutter` (which does make sense to me, just the explanation does not).